src/pages/questionPage.js

```diff
@@ -23,7 +23,16 @@ export const initQuestionPage = () => {
 
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
+    answerElement.id = key;
     answersListElement.appendChild(answerElement);
+    answerElement.addEventListener('click', function () {
+      quizData.currentQuestionAnswer = answerElement.id;
```

Comment:
You don't have to store the `key` as the `id` of the element, I think that you can just use the `key` value also in the event listener.